tests/util/test_dataframe_ops.py

```diff
@@ -52,3 +61,240 @@ def test_parse_check(mock_parse_probtest_csv, setup_csv_files):
 
     pd.testing.assert_frame_equal(df_ref, expected_ref)
     pd.testing.assert_frame_equal(df_cur, expected_cur)
+
+
+def test_force_monotonic():
+    """
+    Test that the function modify the dataframe forcing the values of every line
+    to become non-decreasing monotonic along the columns
+    """
+    # Creation of a DataFrame with MultiIndex on the columns
+    arrays = [
+        ["var1", "var1", "var2", "var2"],
+        ["mean", "max", "mean", "max"],
+    ]
+    columns = pd.MultiIndex.from_arrays(arrays)
+    data = [
+        [1, 5, 2, 7],
+        [3, 2, 1, 9],
+        [2, 8, 5, 4],
+    ]
+    df = pd.DataFrame(data, columns=columns)
+
+    force_monotonic(df)
+
+    # Property verification
+    for stat in df.columns.levels[1]:
+        sub_df = df.loc[:, (slice(None), stat)]
+        assert (sub_df.diff(axis=1).fillna(0) >= 0).all().all()
+
+    # Comparison with expected dataframe
+    expected = pd.DataFrame([[1, 5, 2, 7], [3, 2, 3, 9], [2, 8, 5, 8]], columns=columns)
+    pd.testing.assert_frame_equal(df, expected)
+
+
+def test_compute_rel_diff_basic():
+    """
+    Test that the function is giving the expected values with normal numbers
+    """
+    df1 = pd.DataFrame({"A": [1, 2], "B": [3, 4]})
+    df2 = pd.DataFrame({"A": [1, 1], "B": [3, 5]})
+
+    result = compute_rel_diff_dataframe(df1, df2)
+    expected = pd.DataFrame([[0, 0], [0.33333, 0.2]], columns=["A", "B"])
+
+    pd.testing.assert_frame_equal(result, expected)
+
+
+def test_compute_rel_diff_with_negatives():
+    """
+    Test that the function is giving the expected values also with negative numbers
+    """
+    df1 = pd.DataFrame({"A": [-1, -2], "B": [3, -4]})
+    df2 = pd.DataFrame({"A": [-2, -1], "B": [3, -5]})
+
+    result = compute_rel_diff_dataframe(df1, df2)
+    expected = pd.DataFrame([[0.500, 0], [0.33333, 0.2]], columns=["A", "B"])
+
+    pd.testing.assert_frame_equal(result, expected)
+
+
+def test_compute_rel_diff_with_zeros():
+    """
+    Test that the function is giving the expected values also with zeros in numerator
+    """
+    df1 = pd.DataFrame({"A": [0, 0], "B": [0, 0]})
+    df2 = pd.DataFrame({"A": [1, -1], "B": [2, -2]})
+
+    result = compute_rel_diff_dataframe(df1, df2)
+    expected = pd.DataFrame([[1.0, 2.0], [1.0, 2.0]], columns=["A", "B"])
+
+    pd.testing.assert_frame_equal(result, expected)
+
+
+def test_compute_rel_diff_identical():
+    """
+    Test that the function is giving the expected values aift dataframe are identical
+    """
+    df1 = pd.DataFrame({"A": [1, 2], "B": [3, 4]})
+    df2 = df1.copy()
+
+    result = compute_rel_diff_dataframe(df1, df2)
+
+    assert (result == 0).all().all()
+
+
+def test_compute_division_normal_case():
```

Comment:
To be consistent  with the naming above, I suggest
```suggestion
def test_compute_division_basic():
```